Sources/CustomDump/Conformances/Foundation.swift

```diff
@@ -200,6 +200,37 @@ extension NSTimeZone: CustomDumpRepresentable {
   }
 }
 
+// NB: `NSUserActivity` in unavailable on Linux
+#if os(iOS)
```

Comment:
Should the other OSes be here too?

Reply:
I think this is better since the [NSUserActivity](https://developer.apple.com/documentation/foundation/nsuseractivity) is available on all Apple platforms.
```swift
#if !os(Linux)
```

Reply:
This unfortunately would break Windows, and while we could add an `!os(Windows)` check, too, it would mean potentially breaking other undefined platforms (like WASM), so we think an explicit known platform check is the way to go here.

One complication in the PR is that certain user activity fields are only available on iOS (see "not available on [platform]" comments in the PR diff), but here's an example of how we've selectively introduced things per platform:

https://github.com/pointfreeco/swift-custom-dump/blob/main/Sources/CustomDump/Conformances/UserNotifications.swift#L28-L50